HangPerson/HangPerson/main.m

```diff
@@ -1,18 +1,50 @@
 //
 //  main.m
-//  HangPerson
+//  HangManGame
 //
-//  Created by Michael Kavouras on 6/15/15.
-//  Copyright (c) 2015 Mike Kavouras. All rights reserved.
+//  Created by Jason Wang on 6/17/15.
+//  Copyright (c) 2015 Jason Wang. All rights reserved.
 //
 
 #import <Foundation/Foundation.h>
 
 int main(int argc, const char * argv[]) {
     @autoreleasepool {
+        // Start Code HERE
+        char hangManWord[8] = "zodiacs";
+        char underLineWord[8] = "_______";
         
-        // code goes here...
-
+        int totalLife = 20;
+        char guess;
+        int numGuess;
+        int replace = 7;
+        
+        
+        for (numGuess = 0; numGuess < totalLife; numGuess++) {
+            printf("Please guess a Letter: ");
+            scanf("%c", &guess);
+            for (int i = 0; i < 7; i++) {
```

Comment:
using the number 7 is a bit specific. Instead you could use `strlen(hangManWord)` or `sizeof(hangManWord)`.